src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_vm_diagnostics_templates.py

```diff
@@ -278,248 +278,702 @@ def get_default_diag_config(is_windows):
     else:
         return {
             "ladCfg": {
```

Comment:
Thanks. I suppose this is about the first two Travis CI failures? Added.

I'm looking at the third failure for VMSS, and include a fix for that in the next iteration.